internal/strings/strings_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package strings
+
+import "testing"
+
+func TestMaybeUnquote(t *testing.T) {
```

Comment:
I wonder if we are handled unfinished cases like `"a` or `"'`.

Reply:
This is just a generic helper and not doing validation. Perhaps it should but it was the correct injection point for this code, but currently doesn't have validation semantics so kept like that

Reply:
My point was that in such cases if we don't find a closing quote maybe we want to return the entire string?

Reply:
Ah sorry got it, yeah I think that makes sense will fix